source/administration/identity-access-management/oidc-access-management.rst

```diff
@@ -15,18 +15,64 @@ MinIO supports using an OpenID Connect (OIDC) compatible IDentity Provider (IDP)
 such as Okta, KeyCloak, Dex, Google, or Facebook for external management of user
 identities.
 
-For identities managed by the external OpenID Connect (OIDC) compatible provider, MinIO uses the `JSON Web Token claim <https://datatracker.ietf.org/doc/html/rfc7519#section-4>`__ returned as part of the OIDC authentication flow to identify the :ref:`policies <minio-policy>` to assign to the authenticated user.
+For identities managed by the external OpenID Connect (OIDC) compatible provider, MinIO can use either of two methods to assign policies to the authenticated user.
+
+1. Use the `JSON Web Token claim <https://datatracker.ietf.org/doc/html/rfc7519#section-4>`__ returned as part of the OIDC authentication flow to identify the :ref:`policies <minio-policy>` to assign to the authenticated user.
+2. Use the ``RoleARN`` specified in the authorization request to assign the policies attached to the provider's Role Policy.
+   
 
 MinIO by default denies access to all actions or resources not explicitly allowed by a user's assigned or inherited :ref:`policies <minio-policy>`. 
 Users managed by an OIDC provider must specify the necessary policies as part of the JWT claim. If the user JWT claim has no matching MinIO policies, that user has no permissions to access any action or resource on the MinIO deployment.
 
 The specific claim which MinIO looks for is configured as part of :ref:`deploying the cluster with OIDC identity management <minio-external-iam-oidc>`. This page focuses on creating MinIO policies to match the configured OIDC claims.
 
-Authentication and Authorization Flow
+Authentication and Authorization Flow 
 -------------------------------------
 
+A MinIO cluster can have any number of Role Policy based OpenID providers configured.
+
+A MinIO Cluster may only have **one** JWT Claim based OpenID provider configured.
+Other providers must be configured with the Role Policy method instead.
+
+Role Policy and RoleARN
+~~~~~~~~~~~~~~~~~~~~~~~
+
+MinIO recommends using the Role Policy method for authenticating with an OpenID provider.
+With a Role Policy, all registered users receive the same policy or policies for access.
```

Comment:
```suggestion
With a Role Policy, all clients which generate an STS credential using a given RoleARN receive the :ref:`policy or policies <minio-policy>` associated with that Role Policy configuration.
```

registered users receive the same policy or policies for access.
```

Reply:
Maybe the flow here should be

- Use Role Policies to create an OIDC configuration with a specific set of assigned MinIO Policies

- When you create the configuration, MinIO generates and returns a `RoleARN` for use with the AssumeRoleWithWebIdentity STS API

- Any client who generates credentials using that `RoleArn` and API use those policies for access.

Reply:
Tweaking the suggestion and then updated the flow steps.